fenix/app/src/androidTest/java/org/mozilla/fenix/ui/robots/BrowserRobot.kt

```diff
@@ -590,16 +591,22 @@ class BrowserRobot {
         }
     }
 
-    fun verifyCookiesProtectionHintIsDisplayed(isDisplayed: Boolean) {
-        assertItemContainingTextExists(
-            totalCookieProtectionHintMessage,
```

Comment:
The `totalCookieProtectionHintMessage` variable can be deleted as it's not used anymore.